python/cudf/cudf/tests/test_string.py

```diff
@@ -1591,7 +1587,6 @@ def test_string_starts_ends(data, pat):
     [
         # TODO, PREM: Uncomment after this issue is fixed
         # '',
```

Comment:
Is this case still broken?

Reply:
Yes, I'm trying to see if I can fix the last few of the leftover TODO's by handling in python/cython layer. If not will have to get them resolved in libcudf layer.